test/functional/apps/discover/classic/_doc_table.ts

```diff
@@ -40,6 +40,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.common.navigateToApp('discover');
     });
 
+    after(async function () {
+      await kibanaServer.uiSettings.replace({ 'doc_table:legacy': false });
```

Comment:
One last comment, since `doc_table:legacy` by default is `false`, it should be sufficient to just reset the uiSettings like this, in various `after` functions of the `classic` folder
```suggestion
      await kibanaServer.uiSettings.replace({});
```

Reply:
Done.